Project-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,6 @@
+.container{
+  height: 550px;
+  width: 450px;
+  margin: 0 auto;
+  border: 1px solid grey;
+}
```

Comment:
It's a good idea to set a global style in here adding your box sizing border box property and resetting your margin and  padding to 0